editor/src/themes/questiongroup/QuestionGroup.scss

```
@@ -124,3 +124,59 @@
+
+  .dropdown-menu {
+    border-color: $g-200;
+    border-radius: 4px;
+    padding: 12px;
+    min-width: 220px;
+    box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.2);
+  }
+
+  .dropdown-header {
+    @include label-s-med;
+    text-transform: uppercase;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)